static/src/stylesheets/module/content/_article-minute.scss

```diff
@@ -455,84 +544,49 @@ $minute-layout-margin-nudge: gs-span(1);
     }
 }
 
-/* ==========================================================================
-   Content Overrides
-   ========================================================================== */
-
-.block--minute-article {
-
-    .updated-time {
-        display: none;
-    }
-
-    // Standard elements
-
-    a,
-    .u-fauxlink {
-        color: $us-minute-contrast-colour;
-    }
-
-    a {
-        border-bottom: 1px solid rgba(255, 255, 255, .4);
-        
-        &:hover {
-            border-bottom-color: $us-minute-highlight-colour;
-        }
-
-        &:active {
-            color: $us-minute-highlight-colour;
-            border-bottom-color: $us-minute-highlight-colour;
-        }
-    }
-
-
-    // Captions
-
-    .caption {
-        color: $us-minute-contrast-colour;
-    }
-
-    .svg  & .caption--img:before {
-        @extend %svg-i-camera-white-large;
-        margin-right: 5px;
-    }
+// Background Image Layout
+@supports (object-fit: cover) {
+    .block--background-image {
```

Comment:
Yup, I think you're right. I'll pull these back into the `block--minute-article` scope, was see-sawing on that anyway so no argument from me :)